src/app/games/game/game-faq/game-faq.component.ts

```diff
@@ -0,0 +1,24 @@
+import { KeyValue } from '@angular/common';
+import { ThrowStmt } from '@angular/compiler';
+import { Component, Input, OnChanges, OnInit } from '@angular/core';
+import { forkJoin } from 'rxjs';
+import { GamesService } from 'src/app/games/games.service';
+import { IGame } from 'src/app/games/games.service.interface';
+
+
+ @Component({
+  selector: 'game-faq',
+  templateUrl: './game-faq.component.html',
+  styleUrls: ['./game-faq.component.scss']
+})
+export class GameFAQComponent {
+  
+  constructor(private gamesService: GamesService) { }
+
+  
+  @Input() game: IGame = null;
+
+  onLoad(arg) {}
+  onError(arg) {}
```

Comment:
Maybe a comment explaining why these are blank?

Reply:
I don't know if these are needed any more... are they used? I think the example had them, but didn't do anything with them?

Reply:
I presumed they needed to exist in some form, even if they didn't do anything. If you can safely remove them without error, I'd prefer them gone